### ISSUE TITLE
Updating the RA Autosurgery

### ISSUE THREAD

**Issue by freecitiesbandit** [Free-Cities/Free-Cities#966]
I've noticed that the RA's auto autosurgery options are fairly limited in comparison to the manual player controlled autosurgery. 

Some things that I think should make it over to the RA for ease of automation are: 

1) Areolae shaping and size
2) Labia and clit surgeries
3) Body hair removal
4) The various bone related surgeries

In addition, since the RA now has numerical growth targets it might be worth updating the RA autosurgery to have specific numerical growth targets as well rather than terms such as 'stacked', 'huge' etc.

**Comment by lowercase-donkey** [comment]
Can I add #947, the ability to surgically correct vision?  Such an important detail, and such an odd omission when glasses can be set in RA.

**Comment by Weresmilodon** [comment]
I've toyed with the idea of expanding the RA autosurgery for a while now, but I don't think I have the know-how to do it. I can make some more suggestions though.

* Split apart the cosmetic option so everything within it is optional. Currently it's a whole mess of things, face improvement, liposuction, age lift, breast shape, maybe something more.
* Option to autoreduce breasts, boobs, lips.
* Heeling, both to do and undo.

Basically, most of what is in the normal Surgery should probably be automatable. I'm not entierly sure about the organgrowing though.
